packages/data-context/src/codegen/code-generator.ts

```diff
@@ -116,20 +125,16 @@ function computePath (
 }
 
 async function allFilesInDir (parent: string): Promise<string[]> {
-  let res: string[] = []
+  const dirs = await fs.readdir(parent)
 
-  for (const dir of await fs.readdir(parent)) {
+  const result = await Promise.all(dirs.map(async (dir) => {
     const child = path.join(parent, dir)
     const isDir = (await fs.stat(child)).isDirectory()
 
-    if (!isDir) {
-      res.push(child)
-    } else {
-      res = [...res, ...(await allFilesInDir(child))]
-    }
-  }
+    return isDir ? await allFilesInDir(child) : child
+  }))
 
-  return res
+  return _.flatten(result)
```

Comment:
Are there unit tests that can be associated with this change?

Reply:
There shouldn't be a functional change here, it's just doing what the previous behavior was: 
```
res = [...res, ...(await allFilesInDir(child))]
```
and that should be covered in the unit test suite for `code-generator.spec.ts` which is passing